framework/src/mesh/MooseMesh.C

```diff
@@ -3020,8 +3028,38 @@ MooseMesh::getBoundaryIDs() const
 void
 MooseMesh::buildNodeListFromSideList()
 {
+  auto & boundary_info = getMesh().get_boundary_info();
+
   if (_construct_node_list_from_side_list)
-    getMesh().get_boundary_info().build_node_list_from_side_list();
+  {
+    if (_displace_node_list_by_side_list)
```

Comment:
need to error if displace node list is set when construct_node_list is not since it s ignored

Reply:
Actually - this shouldn't apply if we can default displace_ to true, should it?  If someone's disabling construct_ we don't want their input file to start erroring after this PR.

Reply:
I'll do that in a subsequent PR; in this PR I'm having to leave it disabled by default (Bison fixes are in a PR now, but that'll take time and SAM'll take longer) so I added the error.